tooling/hcpctl/testdata/zz_fixture_TestClusterComponentReleaseSerialization_JSON_serialization.json

```diff
@@ -0,0 +1,46 @@
+{
+  "kind": "ClusterComponentRelease",
+  "apiVersion": "service-status.hcm.openshift.io/v1",
+  "metadata": {
+    "name": "cluster-component-releases",
+    "creationTimestamp": "2025-01-15T12:00:00Z",
+    "aroHcpGithubCommit": "abc123def456",
+    "sdpPipelinesCommit": "xyz789uvw012"
+  },
+  "components": [
+    {
+      "kind": "ComponentRelease",
+      "apiVersion": "service-status.hcm.openshift.io/v1",
+      "metadata": {
+        "name": "backend",
+        "creationTimestamp": "2025-01-15T12:00:00Z"
+      },
+      "workloads": [
+        {
+          "name": "backend-deployment",
+          "namespace": "aro-hcp",
+          "kind": "Deployment",
+          "desiredImage": "myacr.azurecr.io/aro-hcp/backend:v1.2.3",
```

Comment:
We need to be sure we get the digest (SHA) not a tag.  Even if specified via tag, the digests are in status if I recall correctly.

And if we find any tag references, we should keep a list of naughties.